catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/GmtiMetacardType.java

```diff
@@ -37,10 +37,10 @@ public GmtiMetacardType() {
 
     @Override
     public void initDescriptors() {
-        descriptors.addAll(getDescriptors(NitfHeaderAttribute.values()));
```

Comment:
I believe we should be able to get rid of this class and move it into blueprint, since all of its attributes are classes now.